Fabric.Authorization.Persistence.SqlServer/Stores/EDWStore.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Linq;
+using Fabric.Authorization.Domain.Models.EDW;
+using Fabric.Authorization.Domain.Stores;
+using Fabric.Authorization.Persistence.SqlServer.Stores.EDW;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fabric.Authorization.Persistence.SqlServer.Stores
+{
+    public class EDWStore : /*SqlServerBaseStore,TODO EDWServerBaseStore?*/ IEDWStore
+    {
+        private readonly ISecurityContext securityContext;
+
+        public EDWStore(ISecurityContext securityContext)
+        {
+            this.securityContext = securityContext ??
+                    throw new ArgumentNullException(nameof(securityContext));
+        }
+
+        public void AddIdentitiesToRole(string[] identities, string roleName)
+        {
+            var role = this.GetRolesByName(roleName).SingleOrDefault();
+            this.AddIdentitiesToRole(identities, role.Id);
+        }
+
+
+        public void RemoveIdentitiesFromRole(string[] identities, string roleName)
+        {
+            var role = this.GetRolesByName(roleName).SingleOrDefault();
+            this.RemoveIdentitiesFromRole(identities, role.Id);
+        }
+
+
+        private IQueryable<EDWRole> GetRolesByName(string roleName)
+        {            
+            return this.securityContext.EDWRoles.Where(role => role.Name == roleName);
+        }
+
+        private void AddIdentitiesToRole(string[] identityNames, int roleId)
+        {
+            using (var context = this.securityContext.CreateContext())
```

Comment:
We should just be able to use the context, and not have to create a new one for each call and worry about disposing it.